translations/README.md

```diff
@@ -67,16 +71,23 @@ characters?
 `translations: Research translations for <language>.`.
 
 *Completion criteria*: The mentors will verify that you successfully created a
-pull request with at least 6 screenshots of 3 different interfaces and notes.
+pull request with correctly organized screenshots for 3 applications and notes.
+There should be at least 12 screenshots.
```

Comment:
6 screenshots a too few for a multiple-hour task (afaik, the PRs merged last year contained all more than 10 anyway).